scapy/layers/dot11.py

```diff
@@ -206,10 +206,12 @@ class RadioTap(Packet):
                    StrLenField('notdecoded', "", length_from=lambda pkt: pkt.len - pkt._tmp_dissect_pos)]  # noqa: E501
 
     def guess_payload_class(self, payload):
-        if self.Flags.FCS:
-            return Dot11FCS
-        else:
-            return Dot11
+        if self.present and self.present.Flags:
+            if self.Flags.FCS:
+                return Dot11FCS
+            else:
```

Comment:
Unless I'm wrong, this `else` is not needed.

You could:
```Python
def guess_payload_class(self, payload):
    if self.present and self.present.Flags and self.Flags.FCS:
        return Dot11FCS
    return Dot11
```

Reply:
You're right ! I have updated the code.